diff --git a/config.layout b/config.layout
index c42ef2b..d27994f 100644
--- a/config.layout
+++ b/config.layout
@@ -394,3 +394,27 @@
     logfiledir:    ${localstatedir}/log/httpd
     proxycachedir: ${localstatedir}/cache/httpd/proxy
 </Layout>     
+
+# Amazon Linux AMI layout
+<Layout AMZN>
+    prefix:          /usr
+    exec_prefix:     ${prefix}
+    bindir:          ${prefix}/bin
+    sbindir:         ${prefix}/sbin
+    libdir:          ${prefix}/lib
+    libexecdir:      ${libdir}/httpd/modules
+    mandir:          ${prefix}/share/man
+    sysconfdir:      /etc/httpd/conf
+    installbuilddir: ${libdir}/httpd/build
+    includedir:      ${prefix}/include/httpd
+    localstatedir:   /var
+    datadir:         ${localstatedir}/www
+    errordir:        ${datadir}/error
+    iconsdir:        ${datadir}/icons
+    htdocsdir:       ${datadir}/html
+    manualdir:       ${datadir}/manual
+    cgidir:          ${datadir}/cgi-bin
+    runtimedir:      ${localstatedir}/run/httpd
+    logfiledir:      ${localstatedir}/log/httpd
+    proxycachedir:   ${localstatedir}/cache/httpd/cache-root
+</Layout>
